### ISSUE TITLE
Colors broken on HA 2022.12+

### ISSUE THREAD

**Issue by program-the-brain-not-the-heartbeat** [catppuccin/home-assistant#7]
state colors have changed and no longer match this theme.

Forum post: https://community.home-assistant.io/t/2022-12-color-states-are-broken-unusable/499890
Blog post: https://www.home-assistant.io/blog/2022/12/07/release-202212/#shiny-new-state-colors
Developer Blog: https://developers.home-assistant.io/blog/2022/12/07/entity-color-changes/

Variables now have to be defined like:
  state-on-color: var(--rgb-green)
  state-off-color: var(--rgb-red)

  rgb-state-lock-unlocked-color: var(--state-off-color)
  rgb-state-light-color: var(--state-on-color)
  rgb-state-switch-color: var(--state-on-color)

**Comment by GeorgeGedox** [comment]
I don't think this has been fixed yet, the colors look really washed out right now

**Comment by sgoudham** [comment]
Hey 👋,

This repository currently has no maintainers, a PR would be most appreciated.
